src/main/kotlin/org/jitsi/nlj/transform/node/Node.kt

```diff
@@ -258,7 +258,7 @@ sealed class StatsKeepingNode(name: String) : Node(name) {
 
         if (enableStatistics && stats.numInputPackets > 0) {
             synchronized(globalStats) {
-                val classStats = globalStats.computeIfAbsent(name) { NodeStatsBlock(name) }
+                val classStats = globalStats.computeIfAbsent(javaClass.name) { NodeStatsBlock(javaClass.simpleName) }
```

Comment:
What does this do for the anonymous `ConsumerNode`s in `RtpSenderImpl` and `RtpReceiverImpl`?

Reply:
Other anonymous node classes include anything that uses `PipelineBuilder.simpleNode`.